obp-api/src/main/scala/code/api/v3_1_0/OBPAPI3_1_0.scala

```diff
@@ -274,110 +274,9 @@ object OBPAPI3_1_0 extends OBPRestHelper with APIMethods130 with APIMethods140 w
-                          Implementations3_1_0.getMeeting ::
-                          Implementations3_1_0.getServerJWK ::
-                          Implementations3_1_0.createConsent ::
-                          Implementations3_1_0.answerConsentChallenge ::
-                          Implementations3_1_0.getConsents ::
-                          Implementations3_1_0.revokeConsent ::
-                          Implementations3_1_0.createUserAuthContextUpdate ::
-                          Implementations3_1_0.answerUserAuthContextUpdateChallenge ::
-                          Implementations3_1_0.getSystemView ::
-                          Implementations3_1_0.createSystemView ::
-                          Implementations3_1_0.deleteSystemView ::
-                          Implementations3_1_0.updateSystemView ::
-                          Implementations3_1_0.getOAuth2ServerJWKsURIs ::
-                          Implementations3_1_0.getMessageDocsSwagger ::
-                          Implementations3_1_0.updateCustomerEmail ::
-                          Implementations3_1_0.updateCustomerMobileNumber ::
-                          Implementations3_1_0.updateAccount :: 
-                          Implementations3_1_0.addCardForBank :: 
-                          Implementations3_1_0.deleteCardForBank :: 
-                          Implementations3_1_0.getCardForBank :: 
-                          Implementations3_1_0.getCardsForBank :: 
-                          Implementations3_1_0.updatedCardForBank :: 
-                          Implementations3_1_0.updateCustomerIdentity ::
-                          Implementations3_1_0.updateCustomerBranch ::
-                          Implementations3_1_0.updateCustomerCreditLimit ::
-                          Implementations3_1_0.updateCustomerCreditRatingAndSource ::
-                          Implementations3_1_0.updateCustomerData ::
-                          Implementations3_1_0.getMethodRoutings ::
-                          Implementations3_1_0.createMethodRouting ::
-                          Implementations3_1_0.updateMethodRouting ::
-                          Implementations3_1_0.deleteMethodRouting ::
-                          Implementations3_1_0.updateCustomerNumber ::
-                          Implementations3_1_0.createAccount ::
-                          Implementations3_1_0.getPrivateAccountByIdFull ::
-                          Implementations3_1_0.saveHistoricalTransaction ::
-                          Implementations3_1_0.createCardAttribute ::
-                          Implementations3_1_0.updateCardAttribute ::
-                          Implementations3_1_0.getWebUiProps ::
-                          Implementations3_1_0.createWebUiProps ::
-                          Implementations3_1_0.deleteWebUiProps ::
-                          Implementations3_1_0.getBankAccountsBalances ::
-                          Nil
+  val endpointsOf3_1_0 =  getEndpoints(Implementations3_1_0)
```

Comment:
one line method call to collect all endpoints, if you want to exclude given methods. I will explain at OBPAPI4_0_0